.gitignore

```diff
@@ -20,7 +20,6 @@ aggregator/
 tests/.test-reports
 infra/
 .claude
-abi/
```

Comment:
purposely commiting these as they will not change often (ever?) into the future. Solid devEx improvement, always had issues with this in the past